test/e2e/adminactions.go

```diff
@@ -0,0 +1,80 @@
+package e2e
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"crypto/tls"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"os"
+	"strings"
+
+	. "github.com/onsi/ginkgo"
+)
+
+func runAdminTestsInDevOnly() {
+	if os.Getenv("RP_MODE") != "development" {
+		Skip("RP_MODE not set to development, skipping admin actions tests")
+	}
+}
+
+func adminRequest(method string, action string, body string, headers *http.Header, options ...string) ([]byte, error) {
```

Comment:
options should probably be a url.Values

Reply:
function ideally needs some simplification - compare with `func (c *databaseClient) _do`